File: Fibonacci/fib.cpp
#include <iostream>
using namespace std;

// Fibonaccie Series
int Fibonacci(int n) {
    if(n < 2) {
        return n;
    }
    else {
        return Fibonacci(n -1) + Fibonacci(n - 2);
    }
}

// Main driver
int main() {
    int n, res;
    cout << "Enter the number: ";
    cin >> n;
    if(n < 0) {
        return 0;
    }
    else {
        res = Fibonacci(n);
    }
    cout << "The answer is: " << res << "\n";

    return 0;
}

File: Fibonacci/fib.py
# Question

# Function
def Fib(n):
    if n < 2:       # if n is less than 2, then return n
        return n;
    else:
        return Fib(n - 1) + Fib(n - 2)      # recursion
        
n = int(input("Enter the number: "))
print("The number is: ", n)
if n <= 0:
    print("Invalid values.\n")
else:
    res = Fib(n)

print("The answer is: ", res)
